rest/route/service.go

```diff
@@ -70,6 +70,7 @@ func AttachHandler(app *gimlet.APIApp, queue amboy.Queue, URL string, superUsers
 	app.AddRoute("/keys").Version(2).Get().Wrap(checkUser).RouteHandler(makeFetchKeys(sc))
 	app.AddRoute("/keys").Version(2).Post().Wrap(checkUser).RouteHandler(makeSetKey(sc))
 	app.AddRoute("/keys/{key_name}").Version(2).Delete().Wrap(checkUser).RouteHandler(makeDeleteKeys(sc))
+	app.AddRoute("/notifications/{target_id}").Version(2).Post().Wrap(superUser).RouteHandler(makeNotification(environment))
```

Comment:
Is there anything preventing us from allowing all authenticated users to use this route?